lustre-collector/src/quota/quota_parser.rs

```diff
@@ -30,7 +30,7 @@ pub(crate) const QMT_STATS: [&str; 3] = [USR_QUOTAS, PRJ_QUOTAS, GRP_QUOTAS];
 pub(crate) fn params() -> Vec<String> {
     QMT_STATS
         .iter()
-        .map(|x| format!("{QMT}.*.*.glb-{x}"))
+        .map(|x| format!("{QMT}.*.{{mdt,dt}}-*.glb-{x}"))
         .collect()
 }
 
```

Comment:
This is the worst code I have ever seen. I do not see how this is useful. Can you write the parameters out in a static array at the top of the file, and just create a vec out of it in this function ?